docs/pre-release/install/helm.md

```diff
@@ -35,7 +35,7 @@ For more details on what the Helm chart installs and what can be configured, tak
 The Helm chart supports being installed into any namespace, not necessarily `ambassador`. Simply pass a different `namespace` argument to `helm install`:
```

Comment:
I would add a line spelling it out here: "For example, if you wanted to deploy the traffic manager to the `staging` namespace:"